docs/TuxxActionStores.md

```diff
@@ -1,4 +1,5 @@
 # Tuxx ActionStores
+
```

Comment:
Remove the change to this document as it isn't related to this PR.

If you still want to make this change, make a note of it and issue it in a separate PR.